client/src/pages/Review/Review.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import emptyStar from "./../../assets/images/white-background-star.png";
+
+const Review = () => {
+// if input id star-2 is checked 
+let star2 = document.getElementById("2-star")
+
+if (star2) {
+    console.log("star 2 is checked")
+}else {
+    console.log("star2 is not checked")
+}
+
+// then apply gold styles to star-1
+
+
+// if input id star 3 is checked
+
+// then apply gold styles to star-2 and star-1
+
+return(
+    <div>
+        <h1>Review for Business Name Placeholder</h1>
+        <form className="stars">
+            <label>
+                <h3>Rate this restaurant overall:</h3>
+            <input type="radio" name="star-rating" aria-label="One Star" id="1-star" value="1"/>
+            <img src={emptyStar} alt="One Star Rating" />
+            </label>
+
+             <label>
+            <input type="radio"  name="star-rating" aria-label="Two Star" id="2-star" value="2"/>
+            <img src={emptyStar} alt="Two Star Rating" />
+            </label>
+            
+            <label>
+            <input type="radio"  name="star-rating" aria-label="Three Star" id="3-star" value="3"/>
+            <img src={emptyStar} alt="Three Star Rating" />
+            </label>
+
+            <label>
+            <input type="radio"  name="star-rating" aria-label="Four Star" id="4-star" value="4"/>
+            <img src={emptyStar} alt="Four Star Rating" />
+            </label>
+
+            <label>
+            <input type="radio"  name="star-rating" aria-label="Five Star" id="5-star" value="5"/>
+            <img src={emptyStar} alt="Five Star Rating" />
+            </label>
```

Comment:
I will look into a good way to do this star rating the way that you are thinking.

Reply:
Thanks for the react state solution! Added new photos, outlined star and gold star

Reply:
ended up doing a white star instead of an transparent outlined star.